lib/evm/builder.rb

```diff
@@ -86,7 +86,7 @@ def copy(from, to)
 
       private
 
-      def tar_packaged(url,extension)
+      def tar_packaged(url, extension)
```

Comment:
Unrelated

Reply:
Yeah, I've seen people insist on such changes being tacked-on to other work. In future will push separately.